homeassistant/components/bittrex/const.py

```diff
@@ -0,0 +1,23 @@
+"""Constants for Bittrex."""
+from datetime import timedelta
+
+DOMAIN = "bittrex"
+
+# Config
+CONF_MARKETS = "markets"
+CONF_API_SECRET = "api_secret"
+
+# Currency
+CURRENCY_ICONS = {
+    "BTC": "mdi:currency-btc",
+    "ETH": "mdi:currency-eth",
+    "EUR": "mdi:currency-eur",
+    "LTC": "mdi:litecoin",
+    "USD": "mdi:currency-usd",
+}
+
+DEFAULT_COIN_ICON = "mdi:currency-usd-circle"
+
+# Integration setup
+# "In general, making a maximum of 60 API calls per minute should be safe..."
+SCAN_INTERVAL = timedelta(seconds=1)
```

Comment:
This small of a scan interval is incredibly short. Typically cloud pulls of HA integrations are on the order of minutes.

Reply:
Hmm, valid point. But for the cryptocurrency market it's important to update the sensors often. I changed it to `SCAN_INTERVAL = timedelta(seconds=10)` and I'm planning to add it to options so users can change it themselves.